report-ng/app/src/components/method-details/details.ts

```diff
@@ -84,7 +84,10 @@ export class Details {
     }
 
     private _copyStackTraceToClipboard(stackTrace: IStackTraceCause[]) {
-        const msg = stackTrace.flatMap(cause => cause.stackTraceElements).join("\n");
+        let msg = "";
+        stackTrace.forEach(function (value) {
+            msg += value.message + "\n" + value.stackTraceElements.join("\n");
```

Comment:
Does this break the IntelliJ feature *Code* -> *Analyze Stack Trace*?
![Bildschirmfoto 2023-01-06 um 16 45 51](https://user-images.githubusercontent.com/572453/211047008-d5c0149e-81e6-42c1-8110-9c5c3625dc95.jpg)

Reply:
No, it doesn't. IntellJ can handle this. We can also add some formations like indentations to make it more readable.